tests/verkle/eip4762_verkle_gas_witness/test_extcodehash.py

```diff
@@ -157,15 +155,18 @@ def _extcodehash(
 
     post = {}
     if not fails:
-        # TODO(verkle): assign correct storage slot value when filling
         post[TestAddress2] = Account(code=pre[TestAddress2].code, storage={0: 0x424242})
```

Comment:
Looks like post-state checks maybe aren't working?

This `0x424242` is a fake value. Actually should be the `EXTCODEHASH` result saved in storage slot 0 (see L142 above).

So clearly this post-state check should fail. Moreover, you can change to any other value and still pass.

To run: `fill --fork Verkle -v -m blockchain_test -n auto --evm-bin=... -k test_extcodehash_foo`